app/controllers/api/v1/users_controller.rb

```diff
@@ -120,4 +120,24 @@ def update
     standard_update(User, current_human_user.id)
   end
 
-end
\ No newline at end of file
+  ###############################################################
+  # find_unclaimed
+  ###############################################################
+
+  api :POST, '/unclaimed/:email', 'Finds or creates a user by email address.'
```

Comment:
I agree with a more generic endpoint, like /users/find/email
I don't think this is an application_user, since that would mean the user has authorized the app to read their info (which they haven't).